src/components/Gouvernance/NoteDeContexte/ModifierNoteDeContexte.tsx

```diff
@@ -78,16 +74,37 @@ export default function ModifierNoteDeContexte({
   async function modifierUneNoteDeContexte(event: FormEvent<HTMLFormElement>): Promise<void> {
     event.preventDefault()
     setIsDisabled(true)
-    const messages = await modifierUneNoteDeContexteAction({ contenu, path: pathname, uidGouvernance })
-    if (messages[0] === 'OK') {
-      Notification('success', { description: 'modifiée', title: 'Note de contexte ' })
+    if (contenu === '') {
+      const messages = await supprimerUneNoteDeContexteAction({
+        path: pathname,
+        uidGouvernance,
+      })
+      if (messages.includes('OK')) {
+        viderLeContenu()
+        Notification('success', { description: 'bien supprimée', title: 'Note de contexte ' })
```

Comment:
```suggestion
        Notification('success', { description: 'supprimée', title: 'Note de contexte ' })
```
Jsuis repassé sur tous ces messages et j'ai enlevé le "bien" qui n'est pas terrible.